simulation_parameters/microbe_stage/biomes.json

```diff
@@ -164,7 +164,7 @@
         "nitrogen": {
           "Amount": 0,
           "Density": 0,
-          "Ambient": 0.7
+          "Ambient": 0.6
```

Comment:
Is this an intentional change, or just for debugging this? If for just debugging I'd prefer this to be removed when this feature is done.